Financely/Financely/settings.py

```diff
@@ -41,89 +31,62 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'basic_app',
+    'rest_framework',  # New API framework
+    'corsheaders',     # Handle CORS
+    'debug_toolbar',   # Development tool
 ]
 
+# Add new middleware
```

Comment:
**Issue**: Incorrect indentation after control structures

**Reason**: The comment 'Add new middleware' is not indented correctly.

**Suggestion**: Indent the comment by 4 spaces.